tempesta_fw/http.h

```diff
@@ -453,10 +455,14 @@ typedef struct {
 	time_t			date;
 	time_t			last_modified;
 	unsigned long		jrxtstamp;
+	TfwHttpMsg		*req;
 } TfwHttpResp;
 
 #define TFW_HTTP_RESP_STR_START(r)	__MSG_STR_START(r)
 #define TFW_HTTP_RESP_STR_END(r)	((&(r)->body) + 1)
+#define TFW_HTTP_RESP_BOUND_REQ(r)	(TfwHttpReq *)(((TfwHttpResp *)r)->req)
+#define TFW_HTTP_RESP_BIND_REQ(resp, req)	\
+	(((TfwHttpResp *)resp)->req = (TfwHttpMsg *)req)
```

Comment:
First, it appears that there's no need in the outer braces. Secondly, though - it is macro really needed when it's only used once? And then, I am not sure this even makes a good case for a macro...